pera-execution/latest/pera-adapter/src/execution_engine.rs

```diff
@@ -1082,6 +1099,44 @@ mod checked {
         builder
     }
 
+    /// Executes the transaction to store the final MPC output on chain, so it will be accessible to the initiating user.
+    /// All the validators execute this TX locally, and if more than 2/3 of them did so the output is stored on chain.
+    fn setup_and_execute_signature_mpc_output(
+        data: SignatureMPCOutput,
```

Comment:
@cdesaintguilhem can you confirm this is the case?

Reply:
I said it before, there are two points here:
1. In online signing, only one validator computes the signature (the mpc output), the rest just verify it.
2. You don't need 2/3 of validators, but 2/3 of total voting power.

Reply:
@dolevmu the issue of "only one validator computes the signature" is that the MPC code needs to know this; so you would need two different implementations of the last round of the Sign protocol in @ycscaly's code.

Reply:
@ItayLevyOfficial

Reply:
@cdesaintguilhem I did the rename you asked for. This is the right place imo because it's easy for a user to _query the "state of the chain" to access information about the objects that it owns_ with `pera client objects <ADDR>` (pera is the chain's CLI binary).

@dolevmu your first point is irrelevant for here as even if only one validator computes the signature you still need the rest of them to confirm its validity and commit it to the chain, as being done here. Regarding the voting power - here it's actually already being taken into account, my previous documentation was incorrect - I fixed it.